### ISSUE TITLE
"Will you host your code on GitHub? Yes" throws an error

### ISSUE THREAD

**Issue by lguariento** [eXist-db/generator-exist#145]
Hello,
when choosing Will you host your code on GitHub? Yes' I get this error:

```
events.js:167
      throw er; // Unhandled 'error' event
      ^

AssertionError [ERR_ASSERTION]: Trying to copy from a source that does not exist: /usr/local/lib/node_modules/@existdb/generator-exist/generators/app/templates/github/.gitignore
    at EditionInterface.exports.copy (/usr/local/lib/node_modules/@existdb/generator-exist/node_modules/yeoman-generator/node_modules/mem-fs-editor/lib/actions/copy.js:48:3)
    at module.exports.writing (/usr/local/lib/node_modules/@existdb/generator-exist/generators/app/index.js:784:15)
    at Object.<anonymous> (/usr/local/lib/node_modules/@existdb/generator-exist/node_modules/yeoman-generator/lib/index.js:424:27)
    at /usr/local/lib/node_modules/@existdb/generator-exist/node_modules/run-async/index.js:25:25
    at new Promise (<anonymous>)
    at /usr/local/lib/node_modules/@existdb/generator-exist/node_modules/run-async/index.js:24:19
    at self.env.runLoop.add.completed (/usr/local/lib/node_modules/@existdb/generator-exist/node_modules/yeoman-generator/lib/index.js:425:13)
    at runCallback (timers.js:705:18)
    at tryOnImmediate (timers.js:676:5)
    at processImmediate (timers.js:658:5)
Emitted 'error' event at:
    at Immediate.setImmediate (/usr/local/lib/node_modules/@existdb/generator-exist/node_modules/yeoman-generator/lib/index.js:433:22)
    at runCallback (timers.js:705:18)
    at tryOnImmediate (timers.js:676:5)
    at processImmediate (timers.js:658:5)
```

I'm running the latest 1.1.0 version, on OSX 10.11.6.

**Comment by duncdrum** [comment]
note to self the old release did include the `.gitignore` template file according to [travis](https://travis-ci.com/eXist-db/generator-exist/jobs/156682473#L864)

**Comment by duncdrum** [comment]
solved by #147
